### PR TITLE
docs: help documentation link to token URL points to auth.contracts.c.c

### PR DESCRIPTION
**Pull request by blackboxsw**
Fixes: #681

### CODE REVIEW

File: uaclient/cli.py [hunk 1]
```diff
@@ -43,6 +43,7 @@
 Technical support level: {tech_support_level}
 """
 UA_DASHBOARD_URL = 'https://contracts.canonical.com'
```
Comment (OddBloke) [inline uaclient/cli.py]: I think we can drop this constant now, right?

File: uaclient/cli.py [hunk 2]
```diff
@@ -77,7 +78,7 @@ def attach_parser(parser=None):
     parser.add_argument(
         'token', nargs='?',
         help=('Optional token obtained from Ubuntu Advantage dashboard: %s' %
```
Comment (OddBloke) [inline uaclient/cli.py]: Should we still be calling it the "dashboard" in this message?

**Comment by OddBloke** [comment]
Couple of inline questions.

File: uaclient/cli.py (hunk 1 above)
Comment (blackboxsw) [reply uaclient/cli.py]: dropped. nice catch. sloppy of me.

File: uaclient/cli.py (hunk 2 above)
Comment (blackboxsw) [reply uaclient/cli.py]: changed, also fixed the help docs on --otp

**Pull request merged**